core/src/main/scala/Config.scala

```diff
@@ -540,12 +540,16 @@ object Config {
         })
     }
 
-    def readKubernetesInfrastructure(kfg: KConfig): Option[Infrastructure.Kubernetes] = {
-      (kfg.lookup[String]("endpoint") |@| kfg.lookup[Duration]("timeout")) { (endpoint, timeout) =>
+    def readKubernetesInfrastructure(kfg: KConfig): Option[Infrastructure.Kubernetes] =
+      (kfg.lookup[String]("endpoint") |@|
+       kfg.lookup[String]("version").flatMap(KubernetesVersion.fromString) |@|
+       kfg.lookup[String]("caCert").map(p => Paths.get(p)) |@|
```

Comment:
we've avoided camel case for configuration paramaters, in favor of kebab case. Can we rename this to `ca-cert`?

Reply:
kebab-case, love it. will do